### PR TITLE
fix(server): allow glob patterns for section content again and broaden index.styl matching

### PR DESCRIPTION
**Pull request by uglow**
ISSUES CLOSED: #38

fix/38-fix-glob-paths

### CODE REVIEW

File: src/config/webpack/webpack.config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = (config, swankyConfig) => {
 
   swankyConfig.meta.production = process.env.NODE_ENV === 'production';
 
-  var themeStyleRegex = new RegExp(`${swankyConfig.theme}/css/index.styl`);
+  var themeStyleRegex = new RegExp(`${swankyConfig.theme}.*index.styl`);
```
Comment (uglow) [inline src/config/webpack/webpack.config.js]: I know these expressions are not equivalent. I recall seeing some feedback about loosen the requirement for the index.styl file to be within the /css/ subdirectory of the theme.

File: src/config/webpack/webpack.config.js [hunk 2]
```diff
@@ -167,19 +167,19 @@ module.exports = (config, swankyConfig) => {
...
-  // Find content paths
-  swankyConfig.sections.forEach((section) => {
 
+  swankyConfig.sections.forEach((section) => {
+    //console.log('SECTION:', path.extname(section.content), section.content);
     if (section.subSections) {
       section.subSections.forEach((subSection) => {
-        if (path.extname(subSection.content) === 'js') {
-          jsDirs.push(new RegExp(path.join(basepath, subSection.content)));
+        if (path.extname(subSection.content) === '.js') {
```
Comment (uglow) [inline src/config/webpack/webpack.config.js]: This was pretty broken: path.extname() always returns the "."

**Comment by uglow** [comment]
@rodleviton - please review and merge if happy

**Comment by rodleviton** [comment]
@uglow looks good. Merging.

**Pull request merged**